vkubelet/env.go

```diff
@@ -330,6 +332,15 @@ loop:
 		case env.ValueFrom != nil && env.ValueFrom.FieldRef != nil:
 			// TODO Implement the downward API.
```

Comment:
Is this todo done with this change?

Reply:
Yes removed.

Reply:
So can you please remove the TODO as part of this PR?

Reply:
Hi @pires,  will remove it and commit with unit tests. Thanks!